frontend/js/src/user/layout.tsx

```diff
@@ -15,9 +15,13 @@ function NavItem({
 }) {
   return (
     <li
-      className={`${isActive ? "active" : ""} ${isDisabled ? "disabled" : ""}`}
+      className={`nav-item ${isActive ? "active" : ""} ${
+        isDisabled ? "disabled" : ""
+      }`}
     >
-      <NavLink to={url}>{label}</NavLink>
+      <NavLink end className="nav-link" to={url}>
```

Comment:
Can you explain the addition of `end` here?
Not sure what it does

Reply:
We only need to set the class as active when we get the exact match of the endpoint : https://reactrouter.com/6.30.0/components/nav-link#end

Reply:
I see, thanks for the explanation.
Was that just missing previously?

Reply:
Yes, this was missing before, but since we had <li> tag before, the class difference was not visible.